README.md

```diff
@@ -1,13 +1,14 @@
-# Azure SQL bindings for Azure Functions
+# Azure SQL bindings for Azure Functions - Preview
```

Comment:
As a follow up task, remove Preview from any general documentation, and instead just have it for the trigger sections